### PR TITLE
Add subsectors in VATPRC airspace & Adjust boundaries

### LINKED ISSUE #102: ZUGY_CTR missing
image

Controller info: Callsign Guiyang Control CN/EN ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ CPDLC/PDC LOGON GYCT^Â§Cover ZUGY ACC 7800m-GND, ZUGY/AS/PS/BJ/ZY/KJ^Â§China METRIC RVSM in use -> vats.im/rvsm

### LINKED ISSUE #106: ZGKL_CTR missing
image

Callsign Guilin Control ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ï¿½ TEXT ONLY^Â§Cover ZGKL ACC 7800m and below^Â§China RVSM in use - vats.im/rvsm

### PR DESCRIPTION
**Pull request by websterzh**
That resolves the issue in #161 (Hopefully.) But I didn't add Kuching ACC in this PR.
close #102 
close #106 
new

### CODE REVIEW

File: FIRBoundaries.dat [hunk 1]
```diff
@@ -20498,166 +21005,662 @@ ZPKM|0|0|220|21.183333|81.625000|36.416667|109.516667|30.783333|95.570833
...
+29.250075|84.214411
+29.350061|83.914450
+29.216733|83.597828
+29.616689|83.231203
+29.683347|82.997900
+30.016642|82.514628
+30.099961|82.247997
+30.349936|82.164669
+30.441667|81.625000
+30.441667|81.625000
```
Comment (NelisV) [inline FIRBoundaries.dat:21620]: duplicate

File: FIRBoundaries.dat [hunk 2]
```diff
@@ -20498,166 +21005,662 @@ ZPKM|0|0|220|21.183333|81.625000|36.416667|109.516667|30.783333|95.570833
...
+27.700000|98.433333
+27.583333|98.416667
+27.533333|98.350000
+27.966667|98.133333
+28.116667|98.150000
+28.383333|97.900000
+28.333333|97.816667
+28.483333|97.650000
+28.516667|97.583333
+28.516667|97.583333
```
Comment (NelisV) [inline FIRBoundaries.dat:21169]: duplicate

File: FIRBoundaries.dat [hunk 3]
```diff
@@ -20498,166 +21005,662 @@ ZPKM|0|0|220|21.183333|81.625000|36.416667|109.516667|30.783333|95.570833
...
+28.383333|97.116667
+28.316667|97.000000
+28.383333|96.900000
+28.366667|96.775000
+28.416667|96.750000
+28.466667|96.666667
+28.466667|96.583333
+28.416667|96.466667
+28.417089|96.262853
+28.417089|96.262853
```
Comment (NelisV) [inline FIRBoundaries.dat:21417]: duplicate

File: FIRBoundaries.dat [hunk 4]
```diff
@@ -20498,166 +21005,662 @@ ZPKM|0|0|220|21.183333|81.625000|36.416667|109.516667|30.783333|95.570833
...
```
Comment (NelisV) [inline FIRBoundaries.dat:21532]: duplicate

File: FIRBoundaries.dat [hunk 5]
```diff
@@ -20498,166 +21005,662 @@ ZPKM|0|0|220|21.183333|81.625000|36.416667|109.516667|30.783333|95.570833
...
 30.441667|81.625000
 32.301667|84.055000
-ZSHA|0|0|23|23.000000|113.950000|38.250000|124.000000|30.625000|118.975000
+36.416667|87.383333
+36.033333|89.016667
+32.866667|91.916667
+32.100000|97.000000
+31.150000|97.175000
+28.516667|97.583333
+28.516667|97.583333
```
Comment (NelisV) [inline FIRBoundaries.dat:21515]: duplicate

File: FIRBoundaries.dat [hunk 6]
```diff
@@ -20114,7 +20226,216 @@ ZBPE|0|0|70|32.125000|105.229167|45.434694|120.000000|39.984014|112.614584
...
-ZGZU|0|0|123|17.216667|105.541667|29.516667|117.500000|23.366667|111.520834
+ZBAA|0|0|14|36.316667|110.600000|42.250000|120.000000|39.02789|116.643890
+40.100000|113.293333
+40.256944|113.884722
+41.333333|114.516667
+42.185000|116.475000
+42.250000|118.083333
+39.900000|119.350000
+39.500000|119.866667
+39.500000|119.866667
```
Comment (NelisV) [inline FIRBoundaries.dat:20237]: duplicate

File: FIRBoundaries.dat [hunk 7]
```diff
@@ -20856,561 +22286,950 @@ ZWUQ|0|0|271|30.050000|73.600000|49.183333|96.337222|39.616667|84.968611
...
-44.250000|95.000000
-44.283333|95.433333
-44.033333|95.350000
-43.983333|95.550000
-43.400000|95.900000
-43.433333|95.900000
-43.000000|96.166667
-ZYSH|0|0|458|38.000000|115.354167|53.550000|134.766667|45.775000|125.060417
-49.933333|116.600000
+42.216667|80.250000
```
Comment (NelisV) [inline FIRBoundaries.dat:22289]: duplicate

**Review by NelisV — CHANGES_REQUESTED** [review]
Few duplicates, the borders look really nice 👍. 
Although please remove the domestic changes to the russian sectors if not aligned with VATRUS staff.

**Comment by websterzh** [comment]
> Although please remove the domestic changes to the russian sectors if not aligned with VATRUS staff.
We made some changes here just to align the country border. We have noticed the same kind of "gaps" exists as well somewhere there so we did some minor adjustments to eliminate them.

**Comment by NelisV** [comment]
AIRAC 2013 version:
image

This PR:
image

**Comment by websterzh** [comment]
Fixed! I didn't spot that before. Also fixed the missing boundary point in ZMUB.

**Review by NelisV — APPROVED** [review]
Looks good!

**Review by Adrian2k — APPROVED** [review]

**Pull request merged**